{{ cookiecutter.project_slug }}/dev-requirements.txt

```diff
@@ -37,33 +31,30 @@ packaging==24.1
     #   build
     #   pytest
 pip-tools==7.4.1
-    # via datahub (pyproject.toml)
+    # via my_project (pyproject.toml)
 platformdirs==4.3.6
     # via virtualenv
 pluggy==1.5.0
     # via pytest
 pre-commit==4.0.0
-    # via datahub (pyproject.toml)
+    # via my_project (pyproject.toml)
 pyproject-hooks==1.2.0
     # via
     #   build
     #   pip-tools
 pytest==8.3.3
     # via
-    #   datahub (pyproject.toml)
+    #   my_project (pyproject.toml)
     #   pytest-cov
     #   pytest-mock
-    #   pytest-mypy
 pytest-cov==5.0.0
-    # via datahub (pyproject.toml)
+    # via my_project (pyproject.toml)
 pytest-mock==3.14.0
-    # via datahub (pyproject.toml)
-pytest-mypy==0.10.3
-    # via datahub (pyproject.toml)
+    # via my_project (pyproject.toml)
 pyyaml==6.0.2
     # via pre-commit
 ruff==0.6.8
-    # via datahub (pyproject.toml)
+    # via my_project (pyproject.toml)
```

Comment:
This is *really* nit-picky, but I suppose `my_project` could be replaced with `{{ cookiecutter.project_slug }}`

Reply:
I originally did do that, but then I saw that my_project is used in the doc-requirements, so chose the least disruptive option.